app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -113,9 +113,15 @@
 
 .gem-c-feedback__prompt-link {
   @include govuk-font(19);
-  box-shadow: 0 2px 0 govuk-colour("black");
+  background: transparent;
+  box-shadow: 0 2px 0 govuk-colour("white");
+  border: 1px govuk-colour("white") solid;
   min-width: 100%;
 
+  &:hover {
+    background: rgba(govuk-colour("black"), .2);
```

Comment:
We should probably add a fallback hex colour just to be nice to browsers that don't support `rgba`. Not a blocker, since IE11 does support `rgba`.